test/system/500-networking.bats

```diff
@@ -1122,4 +1122,48 @@ EOF
     is "$output" ".*${pasta_iface}.*"
 }
 
+# Refer https://github.com/containers/netavark/issues/1338
+# bats test_tags=ci:parallel
+@test "podman run - dual-stack port binding" {
```

Comment:
That doesn't seem to actually test anything new over existing port forwarding tests. I see nothing here actually testing we bind a dual stack socket. OVerall I don't see why this test would be needed at all when we have the conflict test below